src/lib/rest/rest.cpp

```diff
@@ -915,21 +917,32 @@ bool urlCheck(ConnectionInfo* ciP, const std::string& url)
 *  NO_VERSION: others (invalid paths)
 *
 */
-static ApiVersion apiVersionGet(const char* path)
+static ApiVersion apiVersionGet(const char* path, const std::string& service, const std::string& subService)
 {
+  metricsMgr.add(service, subService, METRIC_TRANS_IN, 1);
```

Comment:
Not sure if the function used to get the API version is, from a logical point of view, the best place to increase the transactions count... For a logical point of view, the best place would be the one in which the incoming transcation is started, just after getting serv and subserv info from headers.